src/drafts.ts

```diff
@@ -9,3 +9,4 @@
 export * from './ActionList2'
 export * from './Button2'
 export * from './ActionMenu2'
+export {default as Label} from './Label2'
```

Comment:
Is there a reason this export doesn't follow the same pattern as the others (i.e `export * from './Label2'`)?

Reply:
It was because `Label` was the default export. I changed it.